docs/index.rst

```diff
@@ -12,9 +12,6 @@ It provides build system independent uploads of source and binary
 `distribution artifacts <distributions_>`_ for both new and existing
 `projects`_.
 
-.. contents:: Table of Contents
-    :local:
-
```

Comment:
Why does this need to be dropped? If the theme doesn't support it, that doesn't seem promising for its overall use.

If we ever get 'round to adding an FAQ, a listing at the top of the page would be useful and the easiest way to do that would be a ToC like this.

Reply:
The theme has its own local ToC on *every page*. Duplicating it in multiple places is unnecessary. This is why when it's present, a red warning is shown so it has to be removed.

If you look at the right sidebar closer, you'll see that it contains all the titles from the local ToC (if you're looking at it on a small screen, you may need to click an icon at the top right corner to reveal it).

The demo for this PR is https://twine--780.org.readthedocs.build/en/780/ — make sure to take a closer look at it.

FWIW its adoption is constantly growing, projects like setuptools, pip, urllib3, attrs, and others switched to Furo months ago. This theme incorporates many modern adaptive web pages techniques making it look great on all sorts of devices (there's probably no themes that make such a great job doing this). It also natively supports adjusting to the dark theme settings of the viewers.